app/controllers/application_controller.rb

```diff
@@ -1,4 +1,31 @@
 class ApplicationController < ActionController::Base
   include AllowBrowser, Authentication, Authorization, SetCurrentRequest, SetPlatform, TrackedRoomVisit, VersionHeaders
   include Turbo::Streams::Broadcasts, Turbo::Streams::StreamName
+
+  before_action :forbid_banned_user
+
+  private
+    def forbid_banned_user
+      return unless Current.user&.banned?
+      return if banned_user_allowed_action?
```

Comment:
We try to avoid early-returns in our code. Could you change this to use a full if or unless statement?